packages/assets-controllers/CHANGELOG.md

```diff
@@ -5,19 +5,52 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [21.0.0]
 ### Added
-- Add `codefiTokenPricesServiceV2` ([#3600](https://github.com/MetaMask/core/pull/3600))
-  - This object can be used for the new `tokenPricesService` argument for TokenRatesController. It uses an internal API to fetch prices for tokens instead of CoinGecko.
+- Add `CodefiTokenPricesServiceV2` ([#3600](https://github.com/MetaMask/core/pull/3600), [#3655](https://github.com/MetaMask/core/pull/3655), [#3655](https://github.com/MetaMask/core/pull/3655))
+  - This class can be used for the new `tokenPricesService` argument for TokenRatesController. It uses a MetaMask API to fetch prices for tokens instead of CoinGecko.
+  - The `CodefiTokenPricesServiceV2` will retry if the token price update fails
+    - We retry each request up to 3 times using a randomized exponential backoff strategy
+    - If the token price update still fails 12 times consecutively (3 update attempts, each of which has 4 calls due to retries), we stop trying for 30 minutes before we try again.
+- Add polling by `networkClientId` to `AccountTrackerController` ([#3586](https://github.com/MetaMask/core/pull/3586))
+  - A new state property, `accountByChainId` has been added for keeping track of account balances across chains
```

Comment:
This is a breaking change since AccountTrackerState is public.

Reply:
Right.... I was considering it non-breaking because it's an addition. But it's a non-optional property, and the state is still a constructor parameter, so I think that makes it breaking. I'll update this

Reply:
Documented here: https://github.com/MetaMask/core/pull/3667/commits/d7d0b01ad8107bd1f63fff8bac56d036e280da12

I left this description under the "Added" entry here because it helps explain what was added, but it's listed separately as a breaking change for anyone reviewing this changelog just for breakages